doc/ref/obsolete.xml

```diff
@@ -199,13 +199,16 @@ Note that <C>MultRowVector</C> was also renamed to <C>MultVectorLeft</C>.
 <InfoClass Name="InfoObsolete"/>
 <Description>
 is an info class to display warnings when an obsolete variable is used.
-By default, these warnings are switched off since the info level for 
-this class is 0. Setting it to 1 will trigger warnings if &GAP; will 
-detect that an obsolete variable is used at runtime.
+By default, the info level for this class is set to 1, which will only
+show variables which will be removed in the next major version of GAP.
+Setting it to 2 will trigger further warnings, for variables which have
+alternative names, or may be removed in future.
+
+This class can be set to 0 to disable all obsolete warnings.
 <P/>
-To check that the &GAP; code does not use obsolete variables at the 
+To check that the &GAP; code does not use any obsolete variables at the
 parsing time, and not at a runtime, use <C>-O</C> command line option,
```

Comment:
Not your fault, so feel free to ignore, but I was wondering:

I am not a native speaker, so this may be totally off, but: "shouldn't it be "at parsing time", without "the"? And then "use -O command line option" -> "use the -O command line option"?

Regardless of that, I wonder how clear the phrase "parsing time" is for the average GAP user. In fact, I am not sure *I* understand what is meant here... Perhaps "... while reading the GAP library"? But even with that... How about: 
> To check that the GAP library and any other GAP code being loaded does not reference or use obsolete variables and functions in any way, one may use the -O command line option (see REF) to completely disable the definition of any obsolete symbols. Note that this may cause various GAP packages to fail to load, or to load incompletely.

Reply:
(but again: I don't require you to make that change, just wondering)

Reply:
I agree with @fingolfin (and that text was written by me, IIRC).